### PR TITLE
Expose advisory Details in API

### PR DESCRIPTION
**Pull request by ykodwani01**
Solves issue #1433 
No change is begin made in models.py still it was showing in files changed
Signed-off-by: Yash Kodwani <yashkodwani2003@gmail.com>

### CODE REVIEW

**Comment by ziadhany** [comment]
@ykodwani01 please add your sign-off DCO is failing.

File: vulnerabilities/models.py [hunk 1]
```diff
@@ -1,1311 +1,1311 @@
...
-        """
-        return cls.objects.log_action(
-            package=package,
-            action_type=PackageChangeLog.FIXING,
-            actor_name=importer,
-            source_url=source_url,
-            related_vulnerability=related_vulnerability,
-        )
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
```
Comment (pombredanne) [inline vulnerabilities/models.py:2]: Why are you changing the models?
Please undo this change.

Comment (ykodwani01) [reply vulnerabilities/models.py:2]: I haven't changed anything still it's showing changes

**Comment by pombredanne** [comment]
>  No change is begin made in models.py still it was showing in files changed

Are you using Git on windows? If so do not translate LF to CR/LF. And please remove the models.py changes

**Comment by ykodwani01** [comment]
New Pull request is made https://github.com/nexB/vulnerablecode/pull/1459
